dist/LICENSE

```diff
@@ -236,7 +237,7 @@ The text of each license is also included at licenses/license-[project].txt.
     k8s.io/kube-openapi v0.0.0-20211115234752-e816edb12b65 Apache-2.0
     k8s.io/utils v0.0.0-20211116205334-6203023598ed Apache-2.0
     sigs.k8s.io/structured-merge-diff/v4 v4.2.1 Apache-2.0
-    skywalking.apache.org/repo/goapi v0.0.0-20240914024804-703f701836e6 Apache-2.0
+    skywalking.apache.org/repo/goapi v0.0.0-20240604102541-64f9001abe03 Apache-2.0
```

Comment:
Why do we need to roll back the go API version? We have some new features replying to the new version.

Reply:
The `dist/LICENSE` file is missing an update.